readgroup.wdl

```diff
@@ -1,68 +1,87 @@
-import "QC/AdapterClipping.wdl" as adapterClipping
-import "QC/QualityReport.wdl" as qualityReport
+version 1.0
+
 import "aligning/align-bwamem.wdl" as wdlMapping
+import "structs.wdl" as structs
 import "tasks/biopet.wdl" as biopet
+import "tasks/common.wdl" as common
+import "QC/AdapterClipping.wdl" as adapterClipping
+import "QC/QualityReport.wdl" as qualityReport
 
-workflow readgroup {
-    Array[File] sampleConfigs
-    String readgroupId
-    String libraryId
-    String sampleId
-    String outputDir
-    Int? numberChunks
+workflow Readgroup {
+    input {
+        Readgroup readgroup
+        String libraryId
+        String sampleId
+        String readgroupDir
+        Int numberChunks = 1
```

Comment:
just to be sure, can the user still set this?

Reply:
yes, this can still be set by the user